### PR TITLE
CSS-9400/add doc for rebac

### PR DESCRIPTION
**Pull request by SimoneDutto**
add documentation for rebac admin

### CODE REVIEW

File: reference/rebac_admin_backend.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+We implement a subset of the operations described in the OpenAPI spec. 
+
+ ====== ============= =========================================================================
+ Status Entities      Notes
+ ====== ============= =========================================================================
+   ✅   capabilities     
+   ✅   entitlements     
+   ✅   capabilities     
+   ✅   groups
```
Comment (ale8k) [inline reference/rebac_admin_backend.rst]: Hehe love these emojicons!!

File: reference/rebac_admin_backend.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The following sections briefly describe the functionalities offered by the ReBAC Admin Backend.
+
+OpenAPI specification
+---------------------
+The OpenAPI spec can be found at this `repository <https://github.com/canonical/openfga-admin-openapi-spec>`__
+
+Authentication
+--------------
+These endpoints are meant to be called from a web browser, therefore the authentication is handle via Cookies.
```
Comment (ale8k) [inline reference/rebac_admin_backend.rst]: Therefor*

**Review by ale8k — APPROVED** [review]

File: reference/rebac_admin_backend.rst (hunk 2 above)
Comment (SimoneDutto) [reply reference/rebac_admin_backend.rst]: is this an english vs american english joke I am too Italian to understand? :laughing:

File: .custom_wordlist.txt [hunk 3]
```diff
@@ -81,6 +85,7 @@ pre
 Quickstart
 RBAC
 ReadMe
+rebac
```
Comment (babakks) [inline .custom_wordlist.txt]: This shouldn't be here.

File: reference/rebac_admin_backend.rst [hunk 4]
```diff
@@ -0,0 +1,32 @@
+ReBAC Admin Backend
+=======================
+
+The following sections briefly describe the functionalities offered by the ReBAC Admin Backend.
+
+OpenAPI specification
+---------------------
+The OpenAPI spec can be found at this endpoint for your deployment: `https://<jimm-deployment>/v1/rebac/swagger.json`
```
Comment (babakks) [inline reference/rebac_admin_backend.rst]: Use double backticks (also notice it's `/rebac/v1/swagger.json`):

```
``https://<jimm-deployment>/rebac/v1/swagger.json``
```

File: reference/rebac_admin_backend.rst [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Authentication
+--------------
+These endpoints are meant to be called from a web browser, therefore the authentication is handled via Cookies.
+
+JAAS Implementation
+-------------------
+
+We implement a subset of the operations described in the OpenAPI spec.
```
Comment (babakks) [inline reference/rebac_admin_backend.rst]: s/We implement/JAAS implements

File: reference/rebac_admin_backend.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
+ReBAC Admin Backend
+=======================
+
+The following sections briefly describe the functionalities offered by the ReBAC Admin Backend.
```
Comment (babakks) [inline reference/rebac_admin_backend.rst]: Please add a sentence or two to explain what ReBAC Admin API is and point to the authorisation docs. As a suggestion:

```
The ReBAC Admin API is a RESTFul API that provides various endpoints to query or 
manipulate relationships in JAAS ReBAC authorisation model.

. hint::
    For an explanation on Relationship-Based Access Control (ReBAC) check out :doc:`this <../explanation/authorisation>` explanation topic.

.. hint::
    As a reference on JAAS authorisation model, check out :doc:`this <../reference/authorisation_model>` reference topic.
```

File: reference/rebac_admin_backend.rst [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+ Status Entities      Notes
+ ====== ============= =========================================================================
+   ✅   capabilities     
+   ✅   entitlements     
+   ✅   capabilities     
+   ✅   groups     
+   ✅   resources     
+   🟡   identities    no support for creation, update and deletion.
+   ❌   roles         no support for roles.
+ ====== ============= =========================================================================
```
Comment (babakks) [inline reference/rebac_admin_backend.rst]: The entries in the second column should be enclosed in double backticks and with a leading slash (`/`) to resemble endpoint realms (e.g., \`\`/capabilitiles\`\`).

File: reference/rebac_admin_backend.rst [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
```
Comment (babakks) [inline reference/rebac_admin_backend.rst]: Keep just one blank at the end.

File: .custom_wordlist.txt [hunk 9]
```diff
@@ -30,9 +31,11 @@ IdP
 init
 installable
 JAAS
+jimm
```
Comment (babakks) [inline .custom_wordlist.txt]: This, too.

File: .custom_wordlist.txt [hunk 10]
```diff
@@ -30,9 +31,11 @@ IdP
...
 JIMM
 JIMM's
 JSON
+json
```
Comment (babakks) [inline .custom_wordlist.txt]: and this.

**Review by babakks — APPROVED** [review]

**Pull request merged**